.drone.yml

```diff
@@ -85,6 +85,17 @@ pipeline:
     when:
       event: [ push, tag ]
 
+  make-rpm:
+    <<: *sbtenv
+    commands:
+      - env JAVAC_VERSION=1.8 sbt clean compile it:compile doc node/rpm:packageBin
+    volumes:
+      - /var/cache/rchain-build/.sbt:/root/.sbt
+      - /var/cache/rchain-build/.ivy2:/root/.ivy2
+      - /var/run/docker.sock:/var/run/docker.sock
```

Comment:
The docker socket mapping belongs to `sbt node/docker:publishLocal` (above).

Reply:
✅